### ISSUE TITLE
Custom Store/Load using a Json object, without using the "Storage" API

### ISSUE THREAD

**Issue by electrotype** [artf/grapesjs#2484]
I'm using the latest version, _0.15.9_, and I'm new to this fantastic project! I'm currently learning a lot, starting from scratch. Please let me know if you prefere such questions on Stack Overflow or elsewhere.

I'm trying to export/import the content by myself, without using the "Storage" API. I see the `store()` function returns the exported content as a JSON object, and this is perfect for my needs! But how can I then programmatically import this JSON  back? I think the `load()` function doesn't take such exported JSON object as a parameter, it forces you to use the "Storage" API, is that correct?

In other words, is there a way to programatically reset the state of a GrapeJs instance using the exported content, as a JSON object?

**Comment by electrotype** [comment]
I think I got it using:

```
let exportToLoad;

let editor = grapesjs.init({
    // ...
    storageManager: { 
        type: 'simpleStorage' 
    }
});

editor.StorageManager.add('simpleStorage', {
    load(keys, clb, clbErr) {
        if(exportToLoad) {
            clb(exportToLoad);
        }
    },
    store(data, clb, clbErr) {
      return data;
    }
});

function importFrom(exportContent) {
    exportToLoad = exportContent;
    editor.load(() => {});
},

```

Then:

`let export = editor.store();`

And:

`importFrom(export);`


That said, it doesn't feel totaly "right" to me. I would like a version of the functions where you can specify the exported Json object directly!

**Comment by artf** [comment]
Using custom storage is actually the best way

> I would like a version of the functions where you can specify the exported Json object directly!

Well, no one limits you to update your exportToLoad
